testing/constraints-3.6.txt

```diff
@@ -5,6 +5,6 @@
 #
 # e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
 # Then this file should have foo==1.14.0
-google-auth==2.13.0
+google-auth==2.14.0
```

Comment:
update real deps as well

Reply:
I guess you are referring to https://github.com/googleapis/google-auth-library-python-oauthlib/blob/main/.kokoro/requirements.txt#L165?

@parthea do you know what is this for and if the google-auth version should be updated here manually?

Reply:
`testing/constraints-3.7.txt` should be testing the minimum versions of dependencies as it says at the top of the file. I'll reply on the PR. So the version in `testing/constraints-3.6.txt` should be `google-auth==2.13.0`. 

The version of `google-auth` in `.kokoro/requirements.txt` is used for the release/docs publication and will be updated automatically as we update dependencies for the release/docs infrastructure.